lib/src/chat_view.dart

```diff
@@ -527,79 +527,87 @@ class DashChatState extends State<DashChat> {
-                                                    .length <=
-                                                3
-                                            ? widget.messages.last.quickReplies
-                                                .values.length
-                                            : 3)
-                                    .map(_mapReply)
-                                    .toList(),
-                              ],
-                            ),
-                    ),
-                  if (widget.chatFooterBuilder != null)
-                    widget.chatFooterBuilder(),
-                  if (!widget.readOnly)
-                    SafeArea(
-                      child: AutoCompleteChatInputToolbar(
-                        key: inputKey,
-                        mentionSuggestionBuilder:
-                            widget.mentionSuggestionBuilder,
-                        getMentionSuggestions: widget.getMentionSuggestions,
-                        sendOnEnter: widget.sendOnEnter,
-                        textInputAction: widget.textInputAction,
-                        inputToolbarPadding: widget.inputToolbarPadding,
-                        textDirection: widget.inputTextDirection,
-                        inputToolbarMargin: widget.inputToolbarMargin,
-                        showTraillingBeforeSend: widget.showTraillingBeforeSend,
-                        inputMaxLines: widget.inputMaxLines,
-                        controller: textController,
-                        inputDecoration: widget.inputDecoration,
-                        textCapitalization: widget.textCapitalization,
-                        onSend: widget.onSend,
-                        user: widget.user,
-                        messageIdGenerator: widget.messageIdGenerator,
-                        maxInputLength: widget.maxInputLength,
-                        sendButtonBuilder: widget.sendButtonBuilder,
-                        text: widget.text != null ? widget.text : _text,
-                        onTextChange: onTextChange,
-                        inputDisabled: widget.inputDisabled,
-                        leading: widget.leading,
-                        trailing: widget.trailing,
-                        inputContainerStyle: widget.inputContainerStyle,
-                        inputTextStyle: widget.inputTextStyle,
-                        inputFooterBuilder: widget.inputFooterBuilder,
-                        inputCursorColor: widget.inputCursorColor,
-                        inputCursorWidth: widget.inputCursorWidth,
-                        alwaysShowSend: widget.alwaysShowSend,
-                        scrollController: widget.scrollController != null
-                            ? widget.scrollController
-                            : scrollController,
-                        focusNode: inputFocusNode,
-                        reverse: widget.inverted,
-                      ),
-                    )
+                  Column(
```

Comment:
if we shove all of this into a column it seems to align correctly.  did you run into problems with this approach @lane-faison ?